torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -284,7 +284,8 @@ struct GraphFuser {
            const char* source,
            const std::string& method_name) {
           script::CompilationUnit cu;
-          cu.define(source, script::nativeResolver, nullptr);
+          cu.define(
+              source, std::make_shared<script::NativeResolver>(), nullptr);
```

Comment:
nit: I don't like std::make_shared, because it always creates bad error messages. Can we make a function `script::nativeResolver()` just return the shared_ptr?